# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../util-vserver/disable_rshared.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

disable rshared mount that according to daniel_hozac is exposing a vanilla problem in umount propagation

vspace --enter builder-urr --index 0 --fs --mount -- cat /proc/mounts

--- ./scripts/vserver.functions.orig	2010-09-29 18:25:31.712000794 +0200
+++ ./scripts/vserver.functions	2010-09-29 18:25:48.260000254 +0200
@@ -975,8 +975,8 @@
     real_vdir=$(getPhysicalDir "$vdir")
     isAvoidNamespace "$cfgdir" || \
 	$_SECURE_MOUNT --rbind -n -o dev "$vdir" "$real_vdir"
-    ! $_VSERVER_INFO - FEATURE PIVOT_ROOT || \
-	$_SECURE_MOUNT -n -o shared,rec "$real_vdir" "$real_vdir"
+#    ! $_VSERVER_INFO - FEATURE PIVOT_ROOT || \
+#	$_SECURE_MOUNT -n -o shared,rec "$real_vdir" "$real_vdir"
 }
 
 function _umountVserverInternal
